frigate/app.py

```diff
@@ -250,36 +250,42 @@ def start_video_output_processor(self) -> None:
     def start_camera_processors(self) -> None:
         model_shape = (self.config.model.height, self.config.model.width)
         for name, config in self.config.cameras.items():
```

Comment:
Similar to previously, instead of the if / else we can just add

```
if not enabled:
  Log
  continue

```

Reply:
Will change. I simply do not like the historic "basic" constructs, with goto, break, continue. Old discussions about readability, workflow etc.